mariadb-cluster/manifests/mariadb-master-rc.yaml

```diff
@@ -0,0 +1,68 @@
+#helm:generate helm tpl -d tpl/values.toml -o manifests/mariadb-master-rc.yaml $HELM_GENERATE_FILE
```

Comment:
The effect is going to be the same as running `helm generate` more than once on a chart.

There could be issues depending on the order the generators are processed. Luckily in this case the one in the `manifest/` directory will be processed before the one in the `tpl/` directory. So there should not be any problems having the generators run multiple times.